packages/@expo/cli/src/start/server/metro/createServerRouteMiddleware.ts

```diff
@@ -109,10 +109,16 @@ export function createRouteHandlerMiddleware(
           }
         }
       },
-      logApiRouteExecutionError(error) {
-        logMetroError(projectRoot, { error });
-      },
-      async handleApiRouteError(error) {
+      async handleRouteError(error) {
+        const { isExpoError } =
+          require('@expo/server/build/error') as typeof import('@expo/server/build/error');
```

Comment:
Should we just re-expose this as an `ExpoError` export and add `ExpoError.isExpoError` as a static method for convenience?